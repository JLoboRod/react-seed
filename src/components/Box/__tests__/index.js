import React from "react";
import { shallow } from "enzyme";

import Box from "../index";

describe("<Box />", () => {
  it("renders correctly", () => {
    const child = <span>I am a child</span>;
    const wrapper = shallow(<Box>{child}</Box>)

    expect(wrapper).toMatchSnapshot();
  });
});