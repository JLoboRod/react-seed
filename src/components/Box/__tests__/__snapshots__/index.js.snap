// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> renders correctly 1`] = `
<div>
  <span>
    I am a child
  </span>
</div>
`;
